Ozair-Ashfaque/Ozair-Ashfaque.md

```diff
@@ -0,0 +1,23 @@
+---
+id: "c5218995-5559-ec11-8f8f-00224815a714"
+
+qualifications: "Master of Science (Network Systems), Bachelor of Computer Science"
+
+quote: "I choose a lazy person to do a hard job. Because a lazy person will find an easy way to do it."
+
+quoteAuthor: "Bill Gates"
+
+---
+
+
+
+Ozair is a Web Developer working at SSW Melbourne with an exprience in web development building responsive and scaleable websites and web applications.
+
+He holds a masters degree of Network System with specialization in Cyber Security from Swinburne University of Technology.He specializes in front-end development using JavaScript frameworks such as Angular and React, and has experience in back-end development using .NET Core.
```

Comment:
change to 

He holds a masters degree of Network Systems with a specialization in Cyber Security from Swinburne University of Technology. He specializes in front-end development using JavaScript frameworks such as Angular and React, and has experience in back-end development using .NET.